index.js

```diff
@@ -77,6 +77,12 @@ Prober.prototype.notok = function notok() {
     }
 };
 
+Prober.prototype.setEnabled = function setEnabled(enabled) {
+    if (typeof enabled === 'boolean') {
```

Comment:
`prober.setEnabled("false")` enables it if we do `!!enabled` or `Boolean(enabled)`. how about we do `assert(typeof enabled === 'boolean)'`?